requirements.txt

```diff
@@ -1,5 +1,5 @@
 future==0.16.0
-six==1.10.0
+six>=1.10.0,<=1.11.0
 dill==0.2.5
 bz2file==0.98
 backports.lzma==0.0.3
```

Comment:
Note that we may consider doing something similar for the latest versions of some other libraries:
-  `dill`: 0.2.7.1
-  `backports.lzma`: 0.0.8
-  `tabulate`: 0.8.1

but if so, I wasn't sure whether you wanted all of that in the same PR, so I just kept the change here minimal.